src/Xamarin.Android.Build.Utilities/Sdks/AndroidSdkWindows.cs

```diff
@@ -15,14 +15,16 @@ class AndroidSdkWindows : AndroidSdkBase
 		const string XAMARIN_ANDROID_INSTALLER_PATH = @"SOFTWARE\Xamarin\MonoAndroid";
 		const string XAMARIN_ANDROID_INSTALLER_KEY = "PrivateAndroidSdkPath";
 
+		string ndkstack = "ndk-stack.exe";
+
 		public override string Adb { get { return "adb.exe"; } }
 		public override string Android { get { return "android.bat"; } }
 		public override string Emulator { get { return "emulator.exe"; } }
 		public override string Monitor { get { return "monitor.bat"; } }
 		public override string ZipAlign { get { return "zipalign.exe"; } }
 		public override string JarSigner { get { return "jarsigner.exe"; } }
 		public override string KeyTool { get { return "keytool.exe"; } }
-		public override string NdkStack { get { return "ndk-stack.exe"; } }
+		public override string NdkStack { get { return ndkstack; } }
```

Comment:
Perhaps instead of fixing up *just* `ndk-stack`, we should fix *all* of then, *and* unify this logic with the Unix logic.

Rephrased, *most* of these properties should *not* be `override`; `AndroidSdkBase` could be handling these.